sys/include/suit.h

```diff
@@ -209,10 +220,6 @@ typedef struct {
  * @brief Bit flags used to determine if SUIT manifest contains components
  */
 #define SUIT_MANIFEST_HAVE_COMPONENTS   (0x1)
```

Comment:
Should this be `SUIT_STATE_%` then?

Reply:
I'm fine with renaming. The difference between the two is scoping. The `SUIT_STATE_*` bit flags are applicable to the whole manifest. The `SUIT_COMPONENT_FLAGS_*` only to specific components.

Reply:
Does `state` need to be a `uint32_t` then? Would you mind grouping all state flags? Same as what you did for component flags?

Reply:
> Does `state` need to be a `uint32_t` then? Would you mind grouping all state flags? Same as what you did for component flags?

Can I leave that first bit out of scope here? It can probably be reduced to `uint16_t`. This is the reason why I wrapped the access for the `suit_component_t::state`, it should be trivial to change it to `uint32_t` if needed